CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added format dictionary field template_suffix.
+
+### Fixed
+- C++ function arguments which passing a class by value.
```

Comment:
passing -> pass ?

Reply:
Also, what was fixed here?  It used to pass by value and now passes by reference?

Reply:
Classes are always passed from Fortran to C via a capsule struct which contains a pointer to the class instance.  The pointer in the capsule needed to be dereferenced when passed to the C++ function in order to pass it by value.  I had fixed this case for POD types, but classes are treated differently since there is no native corresponding type in Fortran.